.github/workflows/integration-tests.yml

```diff
@@ -210,7 +210,7 @@ jobs:
       contents: read
     needs: [build-chainlink, changes]
     if: github.event_name == 'pull_request' && ( needs.changes.outputs.core_changes == 'true' || needs.changes.outputs.github_ci_changes == 'true')
-    uses: smartcontractkit/.github/.github/workflows/run-e2e-tests.yml@f1f2dac0a20f0e02408eb7f528c768fe95c39229
+    uses: smartcontractkit/.github/.github/workflows/run-e2e-tests.yml@ca50645120f0f07ed8c0df08175a5d6b3e4ac454 #ctf-run-tests@0.1.2
```

Comment:
@Tofel why is it called `#ctf-run-tests`?

Reply:
because that's how the tag is called
![image](https://github.com/user-attachments/assets/3c2dbaa1-13b0-477a-a373-84432335dfdc)

Reply:
ok. I guess we could have called it run-e2e-tests. The same as the workflow name.